src/transformers/models/whisper/feature_extraction_whisper.py

```diff
@@ -165,6 +181,7 @@ def __call__(
         max_length: Optional[int] = None,
         sampling_rate: Optional[int] = None,
         do_normalize: Optional[bool] = None,
+        device: Optional[str] = "cpu",
```

Comment:
Could you add this argument to the args docstring, explaining a bit what it does?